MIT.md

```diff
@@ -1,7 +1,4 @@
-## Copyright 2021, [YOUR NAME]
-
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
+## Copyright 2021, Aung Pyae Khant
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```

Comment:
- [ ] Kindly add the current year in this license file instead of 2021 because the license file tells when this project has been created :thinking: You haven't created this project in 2021 so how this project can be licensed in 2021 :smile: So kindly change 2021 to 2023 :+1:
- [ ] Kindly mention the app type. You can add a web app or website :+1: This is your license file so please take it seriously :blush: